app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,7 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="gray_100">#BEEEEEEE</color>
```

Comment:
```suggestion
    <color name="color_gray_100">#BEEEEEEE</color>
```
dùng cái này để biết tên màu: 
https://chir.ag/projects/name-that-color/#6195ED